src/libraries/System.Private.CoreLib/src/System/IO/Path.cs

```diff
@@ -957,11 +957,11 @@ private static string GetRelativePath(string relativeTo, string path, StringComp
             return sb.ToString();
         }
 
+        /// <summary>Gets whether the system is case-sensitive.</summary>
+        internal static bool IsCaseSensitive => PathInternal.IsCaseSensitive;
```

Comment:
I would change the few places that use this to call `PathInternal.IsCaseSensitive` directly instead of introducing this wrapper.